src/controllers/shared.controllers/director.confirm.residential.address.controller.ts

```diff
@@ -19,10 +23,12 @@ export const getDirectorConfirmResidentialAddress = async (req: Request, res: Re
   
     return res.render(templateName, {
       templateName: templateName,
-      backLinkUrl: urlUtils.getUrlToPath(backUrlPath, req),
+      backLinkUrl: addLangToUrl(urlUtils.getUrlToPath(backUrlPath, req), lang),
       directorName: formatTitleCase(directorName),
-      enterAddressManuallyUrl: urlUtils.getUrlToPath(manualEntryUrl, req),
-      ...officerFiling.residentialAddress
+      enterAddressManuallyUrl: addLangToUrl(urlUtils.getUrlToPath(manualEntryUrl, req), lang),
+      ...officerFiling.residentialAddress,
+      ...getLocaleInfo(locales, lang),
+      currentUrl: getCurrentUrl(req, isUpdate, lang),
```

Comment:
I think we can just to `currentUrl = req.originalUrl` - not saying doing this is incorrect.

Reply:
Explicit is better than implicit. This could be flaky if using direct original url, I could visit from a different url

Reply:
> Explicit is better than implicit. This could be flaky if using direct originalUrl, I could visit from a different url

Reply:
Thanks, didn't know. Should we retro-check every page, as I remember doing that on every localisation I have done.?

Reply:
We could, I tried to do a bit of end to end on this ticket as requested by Declan but I guess we could go mop up for code complete on localisation